solr/core/src/java/org/apache/solr/packagemanager/PackageUtils.java

```diff
@@ -140,17 +143,16 @@ public static String getFileFromJarsAsString(List<Path> jars, String filename) {
     return null;
   }
 
-  /** Returns JSON string from a given URL */
-  public static String getJsonStringFromUrl(HttpClient client, String url) {
+  /** Returns JSON string from a given Solr URL */
+  public static String getJsonStringFromUrl(
+      SolrClient client, String path, Map<String, String[]> params) {
```

Comment:
Shouldn't "params" here by SolrParams?  This is pervasive in Solr, and the underlying implementation wants it any way.